tests/unit/stream_alert_rule_processor/test_threat_intel.py

```diff
@@ -76,6 +80,10 @@ def setup(self):
         self.config = load_config('tests/unit/conf')
         self.config['global']['threat_intel']['enabled'] = True
         self.threat_intel = StreamThreatIntel.load_from_config(self.config)
+        os.environ['CLUSTER'] = ''
```

Comment:
Since you always use the `with patch.dict(...` context manager, there is no need to 'reset' this value upon setup here. Can you comment this out and see if the tests still pass? The same may apply for the 2 `reload` statements that follow, but I'm not 100% sure.